macros/edr/system/system_utils/get_config_var.sql

```diff
@@ -48,7 +48,7 @@
     'upload_artifacts_method': 'diff',
     'project_name': none,
     'elementary_full_refresh': false,
-    'min_training_set_size': 14,
+    'min_training_set_size': 7,
```

Comment:
I did this because I noticed that even though `backfill_days` is 2, we only report anomalies for 1 days because the training set size for the day before is 13.
But thought in general that this number should be smaller than `days_back` - so the test can detect anomalies sooner, but will use more data once it is available.

Reply:
just make sure to update the docs, as i think the number 14 is mentioned quite a lot